flink/v1.14/build.gradle

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-String flinkVersion = '1.14.0'
+String flinkVersion = '1.14.4'
```

Comment:
this is not necessary. `iceberg-flink` or `iceberg-flink-runtime` doesn't bring in Flink deps transitively. Flink version is left to the runtime/deployment environment

Reply:
nm. saw other debug logs. you probably just pull in latest version for local testing.

Reply:
nm. saw other debug logs. you probably just pull in latest version for local testing.

Reply:
Correct